ci/after_success.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# Echo all commands before executing them
+set -o xtrace
+# Forbid any unset variables
+set -o nounset
+# Exit on any error
+set -o errexit
+
+# We only need to run the coverage suite once
+COVERAGE_RUN=false
+KCOV_BIN="./target/kcov-master/build/src/kcov"
+
+run_kcov() {
+    # Run kcov on all the test suites
+    if [[ $COVERAGE_RUN != "true" ]]; then
+        cargo coverage --all --tests --benches --no-default-features
+        mv target/kcov target/kcov-no-default-features
```

Comment:
Do we need to move the coverage results like this? The docs for the `cargo coverage` command seem to indicate that it automatically merges all results together into `target/kcov` when run multiple times, but I may be misinterpreting the docs.

Reply:
Unfortunately yes.  `cargo coverage` merges results from one run, but not from several different runs with different features unfortunately.